1-exercises/A-this/exercise-2.js

```diff
@@ -11,9 +11,10 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
-};
-
+    changeAddress: function(newAddress){
+        person.currentAddress = newAddress
```

Comment:
you could also use `this` instead of person as the context is `person` here as in 1-exercises/A-this/exercise-1.js

Reply:
thank you with this also perfectly working